src/YawPitchControl/YawPitchControl.js

```diff
@@ -168,25 +170,28 @@ const YawPitchControl = class YawPitchControl extends Component {
 
 		// adjustOption
 		if (changedKeyList.some(key => key === "yawRange")) {
```

Comment:
@happyhj how about moving this call to _applyOption. and I have other things to talk . let's take meeting.